components/org.wso2.carbon.identity.common/src/main/java/org/wso2/carbon/identity/common/util/IdentityUtils.java

```diff
@@ -92,25 +92,13 @@ public static String generateHmacSHA1(String secretKey, String baseString) throw
      * @return Random hexadecimal encoded String
      * @throws Exception : Exeption
      */
+    /**
+     * generates random string
+     *
+     * @return a random string
+     */
```

Comment:
@Pushpalanka This comment looks ugly. Have a attractive and descriptive comments.